packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## [3.8.1]
+* Added reference in [example documentation](example/README.md) about [writing golden tests for smart widgets with dependency injection](doc/golden_tests.md).
```

Comment:
Considering this md file it will be located at the pub.dev I believe the links need to represented as full URLs such as https://github.com/Prime-Holding/rx_bloc/blob/develop/packages/rx_bloc_cli/example/docs/continuous_delivery.md

Reply:
Updated the links and reorganized how we are including them.